static/usage/v7/datetime/styling/calendar-days/angular/example_component_ts.md

```diff
@@ -0,0 +1,32 @@
+```ts
+import { Component, OnInit, ViewChild } from '@angular/core';
+
+@Component({
+  selector: 'app-example',
+  templateUrl: 'example.component.html',
+  styleUrls: ['example.component.css'],
+})
+export class ExampleComponent implements OnInit {
+  public datetime;
+
+  ngOnInit() {
+    const date = new Date();
+
+    // Set the value of the datetime to 2 days
+    // before the current day
+    let dayChange = -2;
+
+    // If the day we are going to set the value to
+    // is in the previous month then set the day 2 days
+    // later instead so it remains in the same month
+    if (date.getDate() + dayChange <= 0) {
+      dayChange = -dayChange;
+    }
+
+    // Set the value of the datetime to the day
+    // calculated above
+    date.setDate(date.getDate() + dayChange);
+    this.datetime = date.toISOString().substr(0, 19);
```

Comment:
```suggestion
    this.datetime = date.toISOString();
```
Removing the time isn't necessary; the datetime should just ignore it. Ditto for the other frameworks.